src/esrs/postgres/store.rs

```diff
@@ -298,29 +293,23 @@ where
 
         // We need to drop the lock on the aggregate state here as:
         // 1. the events have already been persisted, hence the DB has the latest aggregate;
-        // 2. the policies below might need to access this aggregate atomically (causing a deadlock!).
+        // 2. the event handlers below might need to access this aggregate atomically (causing a deadlock!).
         drop(aggregate_state.take_lock());
 
```

Comment:
Have we checked there are no implications around locking, with the refactor?
I don't see anything obvious but worth double checking

Reply:
You are right. Maybe it's worth to explicit write that in the transactional event handlers is mandatory not to call other aggregates and that are intended to be used for the read side only